src/Agent.Worker/SignatureService.cs

```diff
@@ -30,14 +30,20 @@ public async Task<Boolean> VerifyAsync(Definition definition, CancellationToken
 
             var configurationStore = HostContext.GetService<IConfigurationStore>();
             AgentSettings settings = configurationStore.GetSettings();
-            String fingerprint = settings.Fingerprint;
+            SignatureVerificationSettings verificationSettings = settings.SignatureVerification;
             String taskZipPath = definition.ZipPath;
             String taskNugetPath = definition.ZipPath.Replace(".zip", ".nupkg");
 
             // Rename .zip to .nupkg
             File.Move(taskZipPath, taskNugetPath);
 
-            String arguments = $"verify -Signatures \"{taskNugetPath}\" -CertificateFingerprint {fingerprint} -Verbosity Detailed";
+            String arguments = $"verify -Signatures \"{taskNugetPath}\" -Verbosity Detailed";
+
+            if (verificationSettings?.Fingerprints != null && verificationSettings.Fingerprints.Count > 0)
```

Comment:
Do we want to add this? We are going to deprecate after Skype stops using it and only allow using a CA.

@vtbassmatt Do you want to have both Fingerprints and CA or just Fingerprints?

Reply:
I think it's fine to step away from fingerprints once Skype removes their dependency on it.

Reply:
If you are verifying at all you're always requiring a trusted root. The question is do we also want to support verifying by the specific signing cert's fingerprint, as Nuget allows. The answer is yes, because the agent already does this and we aren't going to take that functionality away. I've essentially just tweaked the representation of that functionality in the config.

Reply:
I think what @vtbassmatt may be saying is that the answer to that is no, once Skype stops using it we don't want to support it any more. Can you clarify, Matt?

Reply:
🤷 I don't think we need it, but Mark wants to hold backcompat. I don't mind either way and I think it's basically his call to make.